src/services/identity/UsersService.ts

```diff
@@ -59,8 +59,11 @@ class UsersService {
     return this.repository.findOne({ where: { githubId } })
   }
 
-  async hasAccessToSite(userId: string, siteName: string): Promise<boolean> {
-    const siteMember = await this.repository.findOne({
+  async hasAccessToSite(
+    userId: string,
+    siteName: string
+  ): Promise<User | null> {
```

Comment:
Hmm I find the return type a little weird. I think we should keep to returning a boolean so that it fits the method name and the purpose of the method (which is to determine if the user has access to the site or not.

Reply:
Hmm in that case can I suggest switching the method name to be something like `getUserIfUserHasAccessToSite` or something like that. I see the merits of saving on boilerplate but I feel based on the naming of the method, I expect to be returned a boolean rather than a User object.

Reply:
i think this is fair but lemme just check w/ @alexanderleegs if he has a downstream dependency here

Reply:
thoughts on calling this `getSiteUser` or `getSiteMember`

Reply:
`getSiteMember` sounds good

Reply:
updated method + call sites